### PR TITLE
Update privacy policy page [MAILPOET-5338]

### PR DESCRIPTION
**Pull request by rodrigoprimo**
## Description

See Jira ticket.

## Code review notes

I checked the uses of the Cookies class and I was not able to find other cookies besides the ones mentioned in the ticket. 

## QA notes

_N/A_

## Linked PRs

_N/A_

## Linked tickets

[MAILPOET-5338]

## After-merge notes

_N/A_

## Tasks

- [x] I followed [best practices](https://codex.wordpress.org/I18n_for_WordPress_Developers) for translations
- [x] I added sufficient test coverage
- [x] I embraced TypeScript by either creating new files in TypeScript or converting existing JavaScript files when making changes

[MAILPOET-5338]: https://mailpoet.atlassian.net/browse/MAILPOET-5338?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by brezocordero — COMMENTED** [review]
@rodrigoprimo 
It looks good. Just one question, do we need to add the form expiration cookies?
https://github.com/mailpoet/mailpoet/pull/3970

**Comment by rodrigoprimo** [comment]
Good catch, @brezocordero. I checked only the PHP code for other cookies, but I failed to check the JS good. I guess we should add this cookie as well (cc @pavel-mailpoet for confirmation).

**Comment by pavel-mailpoet** [comment]
@rodrigoprimo yes please, let's include all cookies we use.

**Comment by rodrigoprimo** [comment]
Thanks for confirming, @pavel-mailpoet.

@brezocordero, I pushed a new commit adding the form cookie and updating the description of the section. I also added another commit with a small formatting change. Can you please check this PR again?

Let me know if I misunderstood how the cookie is used or if you have a suggestion to improve the wording.

File: mailpoet/lib/Config/PrivacyPolicy.php [hunk 1]
```diff
@@ -42,28 +42,44 @@ public function getPrivacyPolicyContent() {
...
+          '<br>' .
+          __('Note: User must be opted-in and a confirmed subscriber.', 'mailpoet') .
+        '</p>' .
+        '<p>' .
+          // translators: %s is the name of the cookie.
+          sprintf(__('Cookie name: %s', 'mailpoet'), 'popup_form_dismissed_{$formId}' ) .
+          '<br>' .
+          __('Cookie expiry: the expiration date varies and can be set per form.', 'mailpoet') .
+          '<br>' .
+          __('Cookie description: This cookie is used to track whether the user has already subscribed to a specific form. It is applicable for popup, slide-in, or fixed bar forms.', 'mailpoet') .
```
Comment (brezocordero) [inline mailpoet/lib/Config/PrivacyPolicy.php]:
```suggestion
          __('Cookie description: This cookie is used to track if a user has previously dismissed a specific form, preventing the re-display of the form until the cookie's expiration date. It is applicable for popup, slide-in, or fixed bar forms.', 'mailpoet') .
```
Something like that? Escaping the apostrophe :)

Comment (rodrigoprimo) [reply mailpoet/lib/Config/PrivacyPolicy.php]: Thanks, Brezo. I added a new commit implementing your suggestion.

**Review by brezocordero — APPROVED** [review]
LGTM!
Thanks for making the changes.

**Review by veljkho — APPROVED** [review]
Not delegating dedicated manual testing to this. Solving high workload by merging without testing. This is more about the text, which can be reviewed within the code changes.

**Pull request merged**